package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "designsystem",
-  "version": "7.1.1",
+  "version": "7.1.2",
```

Bumping Kirby version should only be done when we publish, not as part of implementing a feature.

```suggestion
  "version": "7.1.1",
```